samples/MvcSample.Web/Monitoring/MonitoringMiddlware.cs

```diff
@@ -1,110 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Reflection;
-using System.Text;
-using System.Threading.Tasks;
-using Autofac.Core;
-using Microsoft.AspNet.Builder;
-using Microsoft.AspNet.Http;
-
-namespace MvcSample.Web
-{
-    /// <summary>
-    /// Summary description for MonitoringMiddlware
-    /// </summary>
-    public class MonitoringMiddlware
```

Comment:
I'm sad for us removing the Autofac dependency (temporarily) - but certainly not this from our sample.